src/Sulu/Bundle/MediaBundle/Tests/Unit/Media/ImageConverter/ImagineImageConverterTest.php

```diff
@@ -357,7 +352,7 @@ public function testSupportedOutputFormatsWithValidSvgMimeTypeWithoutSvhImagine(
         $this->assertEquals('png', $this->imagineImageConverter->getSupportedOutputImageFormats($mimeType)[0]);
     }
 
-    public function getSvgMimeTypes(): \Generator
+    public static function getSvgMimeTypes(): \Generator
```

Comment:
test method changes should be done on 2.5 branch if possible and non breaking changes